retail-managment-system/rms-ui/src/app/auth/components/login/login.component.ts

```diff
@@ -69,14 +75,21 @@ export class LoginComponent implements OnInit {
 
          else
          {
-          this._snackBar.open('Email is Not Found!', 'Close', {
+          this._snackBar.open(this.emailError, this.close, {
             duration: 2000,
           });
            return;
          }
     }
   }
 
+  translateError ()
+  {
+    this.emailError= this.translate.instant('logIn.email-error');
+    this.passwordError= this.translate.instant('logIn.password-error');
+    this.close= this.translate.instant('logIn.close');
```

Comment:
you can use  this.translate.instant('logIn.close') and others in place you want without do function for that